pennylane/devices/default_clifford.py

```diff
@@ -0,0 +1,644 @@
+    "SX": "SX",
+    "Adjoint(SX)": "SX_DAG",
+    "CNOT": "CNOT",
+    "SWAP": "SWAP",
+    "ISWAP": "ISWAP",
+    "Adjoint(ISWAP)": "ISWAP_DAG",
+    "CY": "CY",
+    "CZ": "CZ",
+    "GlobalPhase": None,
+    "BasisState": None,
+    "StatePrep": None,
+    "Snapshot": None,
+    "Barrier": None,
+}
+
+
+def operation_stopping_condition(op: qml.operation.Operator) -> bool:
+    """Specifies whether or not an operation is accepted by DefaultClifford."""
+    return op.name in _GATE_OPERATIONS
+
+
+def observable_stopping_condition(obs: qml.operation.Operator) -> bool:
+    """Specifies whether or not an observable is accepted by DefaultClifford."""
+    return obs.name in _MEAS_OBSERVABLES
+
+
+def _import_stim():
+    """Import stim."""
+    try:
+        # pylint: disable=import-outside-toplevel, unused-import, multiple-imports
+        import stim
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires stim, a fast stabilizer circuit simulator. "
+            "It can be installed with:\n\npip install stim"
+        ) from Error
+    return stim
+
+
+class DefaultClifford(Device):
+    r"""A PennyLane device written in Python and capable of executing Clifford circuits using
+    `stim (2021) <https://github.com/quantumlib/stim/tree/main>`_.
+
+    Args:
+        shots (int, Sequence[int], Sequence[Union[int, Sequence[int]]]): The default number of shots to use in executions involving
+            this device.
+        check_clifford (bool): Check if all the gate operations in the circuits to be executed are Clifford. Default is ``True``.
+        max_error (float): The maximum permissible operator norm error for decomposing circuits with non-Clifford gate operations
+            into Clifford+T basis. The default is ``None`` as this device currently supports only Clifford simulations.
+        tableau (bool): Determines what should be returned when the device's state is computed with :func:`qml.state <pennylane.state>`.
+            Default is ``True``, which makes it return the final evolved Tableau. Alternatively, one may make it ``False`` to obtain
+            the evolved state vector. Note that the latter might not be computationally feasible for larger qubit numbers.
+        seed (Union[str, None, int, array_like[int], SeedSequence, BitGenerator, Generator]): A
+            seed-like parameter matching that of ``seed`` for ``numpy.random.default_rng``, or
+            a request to seed from numpy's global random number generator.
+            The default, ``seed="global"`` pulls a seed from NumPy's global generator. ``seed=None``
+            will pull a seed from the OS entropy.
+        max_workers (int): A ``ProcessPoolExecutor`` executes tapes asynchronously
+            using a pool of at most ``max_workers`` processes. If ``max_workers`` is ``None``,
+            only the current process executes tapes. If you experience any
+            issue, try setting ``max_workers`` to ``None``.
```

Comment:
Noticed that these are slightly different to the args text in `default.qubit`. Is this due to different behaviour, or just that the copy is different?